backend/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+	"xkcd/comics/service"
+)
+
+func main() {
+    fmt.Println("Hello, World! Welcome to XKCD go bootcamp")
+
+    latestComicNum, err := service.GetLatestComicNumber()
+    if err != nil {
+        fmt.Printf("Error retrieving latest comic number: %v\n", err)
+        return
+    }
+
+    // Retrieve comics sequentially
+    seqComics, seqDuration := service.RetrieveComicsSequentially(1, latestComicNum)
```

Comment:
Shouldn't be doing both of them at the same time like this. The user should have a way of choosing if they want to do it sequentially or concurrently. 

Also, in other projects, we as devs may want the same, such as being able to use stub data when debugging a service locally. 

Please introduce a flag through which users can specify how they want to run it.